Dockerfile-prod

```diff
@@ -24,12 +24,10 @@ COPY .env* ./
 # Building the image
 RUN npm run build
 
-# Copy env files
```

Comment:
Why was this removed?

Reply:
Hmm, this seems to be the old wrongly placed one, but was this not already removed?

Reply:
I did a git pull on develop before this, so it must have slipped, it's no big deal actually